synapse/config/password.py

```diff
@@ -48,4 +52,29 @@ def generate_config_section(self, config_dir_path, server_name, **kwargs):
            # DO NOT CHANGE THIS AFTER INITIAL SETUP!
            #
            #pepper: "EVEN_MORE_SECRET"
+
+           # Define and enforce a password policy. Each parameter is optional, boolean
+           # parameters default to 'false' and integer parameters default to 0.
+           # This is an early implementation of MSC2000.
+           #
+           #policy:
+              # Whether to enforce the password policy.
+              #
+              #enabled: true
```

Comment:
Could you add spaces between each item here (see the below config block)? It's currently a bit difficult to read.

Reply:
to add to @anoadragon453 's comments: please remember to explicitly document the defaults.

Reply:
oh I see that the defaults are documented further up; I missed them there. I think they should be documented next to the options in question.

Reply:
Yeah, also allows for different defaults for different policy types, which may be useful in the future.